plugins/woocommerce/includes/emails/class-wc-email-new-order.php

```diff
@@ -165,6 +165,18 @@ public function get_default_additional_content() {
 			return __( 'Congratulations on the sale.', 'woocommerce' );
 		}
 
+		/**
+		 * Return content from the additional_content field.
+		 *
+		 * Displayed above the footer.
+		 *
+		 * @since 3.7.0
+		 * @return string
+		 */
+		public function get_additional_content() {
+			return apply_filters( 'woocommerce_email_additional_content_' . $this->id, $this->format_string( $this->get_option( 'additional_content' ) ), $this->object, $this ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
```

Comment:
Thinking that rather than suppress the linter warning, which makes it a little harder to surface missing docblocks in the future (because PHPCS will stop complaining about any with this `phpcs:ignore` comment), probably the best course of action would be to document the original hook in `WC_Email` and then [mark this one as a duplicate hook](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-1-duplicate-hooks).

Reply:
@barryhughes I was discussing this with @jorgeatorres and he mentioned using the `phpcs:ignore` comment because it was a duplicate.  Maybe he can jump in to this conversation and give his opinion.